mapbox/vs30/scripts/basin_z_values/gen_sites_in_basin.py

```diff
@@ -0,0 +1,160 @@
+from pathlib import Path
+import time
+import argparse
+
+from numba import jit, njit
+import numba
+import numpy as np
+import pandas as pd
+from Velocity_Model.basins import basin_outlines_dict
+from qcore import geo, formats
+import pygmt
+
+GRID_SPACING = 100
+BASIN_VER = "2.07"
+
+
+@jit(nopython=True)
+def is_inside_postgis(polygon, point):
+    length = len(polygon)
+    intersections = 0
+
+    dx2 = point[0] - polygon[0][0]
+    dy2 = point[1] - polygon[0][1]
+    jj = 1
+
+    while jj < length:
+        dx = dx2
+        dy = dy2
+        dx2 = point[0] - polygon[jj][0]
+        dy2 = point[1] - polygon[jj][1]
+
+        F = (dx - dx2) * dy - dx * (dy - dy2)
+        if 0.0 == F and dx * dx2 <= 0 and dy * dy2 <= 0:
```

Comment:
For this sort of computational code, we should directly compare to zero, either `np.isclose(F, 0)` or `abs(F) < tol` would be preferable here.